astro/src/components/listing/entry.astro

```diff
@@ -4,13 +4,30 @@ export interface Props {
 }
 
 const report = Astro.props.report
+
```

Comment:
@iam-benyamin we need to use a multilanguage library for this, as we need to display both absolute date, in format for the language we are targeting, and relative date.
Example


Input: 2023-05-01

Output (EN) May 5th 2023 ( 23 days ago )
Output (IT) 5 Maggio 2023 ( 23 giorni fa ) 

For these cases, i'd suggest to use Moment js or something similar